FogCreek/Glitch-Community
# Quixotic rocket: Only send the experiment viewed event if the experiment is active

## Links
https://quixotic-rocket.glitch.me/

## Changes:
The 'Experiment Viewed' event doesn't get sent until the experiment is actually enabled

## How To Test:
Try disabling a feature in optimizely (this remix is using the dev optimizely config) and check that the segment.io/t request is not sent. It should be sent if the feature is enabled for any percent of users, even if it isn't active for you in particular
